src/physics/base_solver.cpp

```diff
@@ -118,6 +118,20 @@ void BaseSolver::initializeOutput(const serac::OutputType output_type, const std
       break;
     }
 
+    case serac::OutputType::ParaView: {
```

Comment:
Does this require any extra dependencies?

Reply:
No, its self contained in mfem. It just outputs text through a stream.